RELEASE.md

```diff
@@ -20,6 +20,7 @@ Please follow the established format:
 - Move data source loading into standalone-app entry point (#215)
 - Allow an argument to be passed to loadJsonData (#215)
 - Add information for multiple pipelines in `nodes.json` (#192)
+- Rename default endpoint from `/api/nodes.json` to `/api/nodes` (#239)
```

Comment:
Not entirely sure if this is major or minor change 🤔 

It also looks like @921kiyo 's changes in https://github.com/quantumblacklabs/kedro-viz/pull/231 aren't added here?

Reply:
I'm inclined to say major but not necessarily breaking (though it's skirting that line as people will need to rename their test json files)

Reply:
Hi, I'm back! 
I didn't add the change https://github.com/quantumblacklabs/kedro-viz/pull/231 in release note because it's a back-end change. If we expect new API endpoints to be used by users (as a React component I guess?), then it makes sense to add it in the release note? Otherwise probably not necessary.

Reply:
ah okay! I wasn't sure if release notes were only for users or also for ourselves to keep track of the changes we've made 🙂

Reply:
@921kiyo I don't expect the extra endpoints to be used in the React component library, as users would need to provide their own endpoints. Instead, I think that if they want to provide this data, they can include it in the main data prop. Haven't discussed this much but idk how it could work otherwise.

@MerelTheisenQB A bit of both, but I see the release notes as mainly for users.